pkg/extension/init/init.go

```diff
@@ -102,7 +102,12 @@ func (s *scaffolder) walk(sourceDirName, destinationDirName string) (errs error)
 }
 
 func (s *scaffolder) visit(sourceDirName, destinationDirName string, sourceFileInfo os.FileInfo) (errs error) {
-	relOutputFileName := filepath.Join(destinationDirName, sourceFileInfo.Name())
+	baseOutputFileName := sourceFileInfo.Name()
+	// We rename go.mod to go.mod_ to workaround https://github.com/golang/go/issues/45197
```

Comment:
this is far far easier than writing an FS renamer, especially considering we are already walking the file tree. However, I linked the issue as it is soliciting a library to do that.